src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.ReinferredVariableTracker.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal sealed partial class NullableWalker
+    {
+#nullable enable
+        internal sealed class ReinferredVariableTracker
```

Comment:
>ReinferredVariableTracker [](start = 30, length = 25)

Is this class still used? Remove the file?

Reply:
It is not used, apparently when I added I changes I missed adding the delete.

---
In reply to: [329706888](https://github.com/dotnet/roslyn/pull/38769#discussion_r329706888) [](ancestors = 329706888)